File: java/src/test/java/ru/kontur/courses/homework/Person.java
package ru.kontur.courses.homework;

public class Person {
    public static int idCounter = 0;
    public int age, height, weight;
    public String name;
    public Person parent;
    public int id;

    public Person(String name, int age, int height, int weight, Person parent) {
        id = idCounter++;
        this.name = name;
        this.age = age;
        this.height = height;
        this.weight = weight;
        this.parent = parent;
    }
}

File: cs/HomeExercises/HomeExercises/Person.cs
namespace HomeExercises;

public class Person
{
    public static int IdCounter;
    public int Age, Height, Weight;
    public int Id;
    public string Name;
    public Person? Parent;

    public Person(string name, int age, int height, int weight, Person? parent)
    {
        Id = IdCounter++;
        Name = name;
        Age = age;
        Height = height;
        Weight = weight;
        Parent = parent;
    }
}